msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/errors/SignUpErrors.kt

```diff
@@ -38,11 +38,6 @@ internal class SignUpErrorTypes {
          */
         const val USER_ALREADY_EXISTS = "user_already_exists"
 
-        /* The INVALID_USERNAME value indicates the username provided by the user is not acceptable to the server.
```

Comment:
So we're combining the SDK generated invalid username error with the API generated one? I'm in favour of that, but this requires a cleanup in common: `SignUpCommandResult.InvalidEmail` result.

Reply:
I have cleaned it up.